addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2206,7 +2207,8 @@ export class OdooEditor extends EventTarget {
     _positionToolbar() {
         const OFFSET = 10;
         let isBottom = false;
-        this.toolbar.classList.toggle('toolbar-bottom', false);
+        // Toolbar display must not be none in order to calculate width and height.
```

Comment:
Ok, but does that have something to do wit the `toolbar-bottom` class toggle that was there before ?

Reply:
No. It has to do with the line below. If the toolbar has display none,  toolbarWidth and toolbarHeight (some 15 lines below) will have 0 as value. 

I placed this (`this.toolbar.classList.toggle('d-none', false);`)  here at the top of the function because it is where the other resets are.
But I could move it to right before it is needed, namely above
```
        const toolbarWidth = this.toolbar.offsetWidth;
        const toolbarHeight = this.toolbar.offsetHeight;
```
What do you think?

Reply:
The reason why `this.toolbar.classList.toggle('toolbar-bottom', false);` was removed is because it is no longer needed: see [this] comment.

[this]: https://github.com/odoo/odoo/pull/123918/files/19e9fea7abc109843deff72ca446bb9466a5ca9a..c1aa8d98489f18fe2596e08aabe4059850831956#r1239838158